### ISSUE TITLE
Park doesn't seem to correctly set and return parked property for client

### ISSUE THREAD

**Issue by EorEquis** [ClutchplateDude/OpenAstroTracker#17]

**Comment by EorEquis** [comment]
This is a timeout issue.  Since park returns 1 only once the mount is parked, we need to handle longer wait times.

This will impact Synced slews as well.

`08:41:15.442 CommandString             Transmitted :hP#
08:41:16.057 CanSlewAsync              Get - True
08:41:17.444 CommandString(:hP#)       Timed out waiting for received data
08:41:17.444 CommandString             Transmitted :GR#
08:41:19.446 CommandString(:GR#)       Timed out waiting for received data
08:41:19.446 RightAscensionGet         255`

**Comment by EorEquis** [comment]
Property is now being set correctly.   I have some more work to do testing the new "async but synced" park, but it seems like the original issue is fixed.
